vumi/transports/smpp/smpp_transport.py

```diff
@@ -499,6 +504,12 @@ def _check_stop_throttling(self):
         """
         self._unthrottle_delayedCall = None
 
+        if not self.service.is_bound():
+            # We don't have a bound SMPP connection, so try again later.
+            log.msg("Can't check throttling while unbound, trying later.")
+            self.check_stop_throttling(self.get_static_config().throttle_delay)
```

Comment:
If that is the case, maybe we should mention something about how it is used when we are not yet bound in [the config field's description](https://github.com/praekelt/vumi/blob/feature/issue-909-smpp-throttling-reconnect/vumi/transports/smpp/config.py#L20-L22)?